images/pod/Dockerfile.centos7

```diff
@@ -22,6 +22,6 @@ RUN INSTALL_PKGS="origin-pod" && \
     ln -s /usr/bin/pod /pod
```

Comment:
That line is/was only in the Dockefile.centos7 because the ENTRYPOINT is/was "/pod".  If the ENTRYPOINT in the normal Dockerfile is changed, then it should be changed in Dockerfile.centos7, and the "ln -s /usr/bin/pod /pod" line should be taken out of Dockerfile.centos7